### ISSUE TITLE
Remove GHC start-up warnings

### ISSUE THREAD

**Issue by justb4** [geopython/GeoHealthCheck#137]
When GHC starts there are several warnings. Though harmless, these could confuse users. In particular these warnings should be removed:

```
GeoHealthCheck/app.py:38: ExtDeprecationWarning: Importing flask.ext.babel is deprecated, use flask_babel instead.
  from flask.ext.babel import Babel, gettext
GeoHealthCheck/app.py:39: ExtDeprecationWarning: Importing flask.ext.login is deprecated, use flask_login instead.
  from flask.ext.login import (LoginManager, login_user, logout_user,
GeoHealthCheck/init.py:32: ExtDeprecationWarning: Importing flask.ext.sqlalchemy is deprecated, use flask_sqlalchemy instead.
  from flask.ext.sqlalchemy import SQLAlchemy
 * Running on http://0.0.0.0:8001/ (Press CTRL+C to quit)
 * Restarting with stat
GeoHealthCheck/app.py:38: ExtDeprecationWarning: Importing flask.ext.babel is deprecated, use flask_babel instead.
  from flask.ext.babel import Babel, gettext
GeoHealthCheck/app.py:39: ExtDeprecationWarning: Importing flask.ext.login is deprecated, use flask_login instead.
  from flask.ext.login import (LoginManager, login_user, logout_user,
GeoHealthCheck/init.py:32: ExtDeprecationWarning: Importing flask.ext.sqlalchemy is deprecated, use flask_sqlalchemy instead.
```

And

```
SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True to suppress this warning.
```
